modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/InstrumentControllerSim.scala

```diff
@@ -3,37 +3,45 @@
 
 package edu.gemini.seqexec.server
 
-import java.util.concurrent.atomic.AtomicBoolean
+import java.util.concurrent.atomic.{AtomicBoolean, AtomicInteger}
 
 import edu.gemini.seqexec.model.dhs.ImageFileId
 import edu.gemini.seqexec.server.SeqexecFailure.SeqexecException
 import gov.aps.jca.TimeoutException
 import org.log4s.getLogger
 import squants.Time
+import squants.time.Seconds
 
 import scala.annotation.tailrec
 import scalaz.{EitherT, Show}
 import scalaz.concurrent.Task
 import scalaz.syntax.show._
 import scalaz.syntax.std.boolean._
 
-class InstrumentControllerSim(name: String) {
+class InstrumentControllerSim(name: String, useTimeout: Boolean) {
   private val Log = getLogger
 
   private val stopFlag = new AtomicBoolean(false)
   private val abortFlag = new AtomicBoolean(false)
   private val pauseFlag = new AtomicBoolean(false)
+  private val remainingTime = new AtomicInteger(0)
 
   private val tic = 200
 
+  private val ReadoutDelay = Seconds(5)
```

Comment:
This is not important but with the right squants import you can write `5.seconds`